fuzzing/Fuzzing.md

```diff
@@ -0,0 +1,62 @@
+#####
+How-to Guide for fuzzing in localhost.
+
+
+- Export flags.
+
+####
+LibFuzzer with AddressSanitizer
+
+```
+export CC=clang
```

Comment:
Thinking out loud: having these 3 options in a script (or maybe as separate makefile targets) to make it easier to run the fuzzer/automate it (when run not a part of a fuzzing cluster which I suppose injects those settings)?